### DISCUSSION TITLE
dialog导致页面抖动

### DISCUSSION THREAD

**Discussion by ghost** [tusen-ai/naive-ui#3994] [Q&A]
打开dialog会导致页面其他元素发生抖动。求解

**Comment by XieZongChen** [comment]
需要一些上下文信息，通常情况下这是因为受到一些样式影响，请检查使用处的上下文。

**Reply by ghost** [reply]
只是引用了官网的一些demo。测试样例，并没有做什么工作。
![VeryCapture_20221106184125](https://user-images.githubusercontent.com/32454224/200167181-d6bc7505-7fae-4b46-91c7-927c08e77049.gif)

**Reply by ghost** [reply]
> 需要一些上下文信息，通常情况下这是因为受到一些样式影响，请检查使用处的上下文。

看起来像是弹窗把滚动条撑没了，导致页面自适应的样子。
据我所知elementplus和arco design都有这个现象

**Reply by XieZongChen** [reply]
这是为了避免弹出组件出现滚动条，应该是正常现象

**Reply by XieZongChen** [reply]
如果接受不了可以全局把滚动条样式修改窄一点，应该会好一些
